src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -130,7 +129,7 @@ class ReportActionContextMenu extends React.Component {
             {
                 text: this.props.translate('reportActionContextMenu.deleteComment'),
                 icon: Trashcan,
-                shouldShow: () => this.canEdit(),
+                shouldShow: this.canEdit(),
```

Comment:
This isn't going to work because it will execute `canEdit()` as soon as this line is hit (rather than when `shouldShow` is referenced).